src/client/ClientDataResolver.js

```diff
@@ -244,7 +244,15 @@ class ClientDataResolver {
         if (/^https?:\/\//.test(resource)) {
           request.get(resource)
             .set('Content-Type', 'blob')
-            .end((err, res) => err ? reject(err) : resolve(res.body));
+            .end((err, res) => {
```

Comment:
If I drop the `else`, won't that mean we get both a `reject` and a `resolve` call in case `err` is truthy?